<launch>

    <node pkg="tf" type="static_transform_publisher" name="camera_link" args="-2.11 3.31 -1.28 1.5707 0 1.5707  /map /camera_link  100"/>
    <node pkg="yolo_tag_detector" type="yolo_node.py" name="yolotag_node" output="screen" >
        <rosparam command="load" file="$(find yolo_tag_detector)/config/yolotag_param_new.yaml" />
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find yolo_tag_detector)/config/yolo.rviz" />\
</launch>